src/docs/guide/interactivity.md

```diff
@@ -7,4 +7,4 @@ order: 7
 show_guide: true
 ---
 
-Coming soon.
+Coming soon. See the [cursor component docs](../components/cursor.html) in the meantime.
```

Comment:
a bit minor, but don't think we need to backtick component names. i think we can intend "the material component" to read like english. esp. since we're going to use them very often. don't want to clutter with `<code>`s